src/globals/config/types.ts

```diff
@@ -54,6 +54,7 @@ export type PayloadGlobalConfig = {
   }
   fields: Field[];
   admin?: {
+    description?: string,
```

Comment:
This should also be `string | (val: unknown) => string | React.ComponentType` AFAIK